### DISCUSSION TITLE
Patch Panels?

### DISCUSSION THREAD

**Discussion by opticalmatrix** [netbox-community/netbox#7371] [Q&A] [answered]
Hi all,

Been using Netbox for about a month now for work and the only thing I haven't been able to map out are the fiber/copper patch panels that we have. I feel like that would be an advantage to tracing issues within the network, am I missing where these are or is there a add-on/plug-in that I might be missing?

Thanks!

**Comment by jeremystretch** [comment, marked as answer]
Patch panels in NetBox are just modeled as devices, same as the rest. You can add front/rear "pass-through ports" to model passive connections through the panels. Multiple front ports can map to a single rear port to model e.g. MPO/MTP fiber, etc.

**Reply by opticalmatrix** [reply]
Great thanks!
